src/App.js

```diff
@@ -1,25 +1,99 @@
-import logo from './logo.svg';
+export default App;
 import './App.css';
+import { useState } from 'react';
+import MovieForm from './MovieForm.js';
+import { useEffect } from 'react';
+
+
+
+
+
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+  
+  const [title, setMovieTitle] = useState('StarWars');
+  const [director, setMovieDirector] = useState('George Lucas');
+  const [year, setMovieYear] = useState('1979');
+  const [color, setMovieColor] = useState('lightgrey');
+  // const [moviesList, setMoviesList] = useState({});
+  
+  const [allMovies, setMovies] = useState([]);
+  const [visibleMovies, setVisibleMovies] = useState(allMovies);
+  
+
+
+
+  function handleSubmit(e) {
+    e.preventDefault();
+  
+    const movie = {
+      title: title,
+      director: director,
+      color: color,
+    };
+
+    const updatedMovies = [...allMovies, movie];
+    setMovies(updatedMovies);
```

Comment:
here, you can use the state-setting functions you declared for each of the input fields to clear out the form on submission